.travis.yml

```diff
@@ -36,3 +36,4 @@ install:
   
 script:
   - pytest
+  - mypy ppb_vector tests
```

Comment:
I think you need to set `MYPYPATH` to be `${VIRTUAL_ENV}`, otherwise `mypy` will complain about being unable to resolve the `hypothesis` imports.

Reply:
IS THAT WHAT"S GOING ON?????????????

Reply:
Except the build isn't using virtual environments.

Reply:
Yeah, this is only a thing that's happening locally.